Cargo.toml

```diff
@@ -21,3 +21,5 @@ reqwest = { version = "0.11.12", features = ["blocking", "json"] }
 ureq = "2.5.0"
```

Comment:
Let's remove this unused library, I think cargo builds everything from dependency or something